lwcomponent/catalog.go

```diff
@@ -259,13 +259,23 @@ func NewCatalog(client *api.Client, stageConstructor StageConstructor) (*Catalog
 
 	cdkComponents := make(map[string]CDKComponent, len(rawComponents)+len(localComponents))
 
+	allVersions := []semver.Version{}
+
+	for _, c := range rawComponents {
```

Comment:
This doesn't have all of the available versions.  We need to call API `ListComponentVersions`.

Like in https://github.com/lacework/go-sdk/blob/main/lwcomponent/catalog.go#L80

An API call for every component installed.  Thankfully we will cache this.